src/_includes/components/navigation-secondary.njk

```diff
@@ -1,39 +1,25 @@
 {% set localeParentKey = '/' + locale + '/' %}
 {% set primaryNav = collections.primary | eleventyNavigation(localeParentKey) %}
-{% set secondaryNav = primaryNav[0].children %}
-
-{# TEMP #}
-<style>
-  [aria-label="secondary"] ul {
-    list-style: none;
-    margin: 0;
-    padding: 0;
-  }
-  [aria-label="secondary"] li {
-    display: inline-block;
-    margin-top: 0.5em;
-  }
-  [aria-label="secondary"] [aria-current="page"] {
-    background-color: #ddd;
-  }
-</style>
 
 {# Not sure about this if layout != home as it feels a bit flimsy #}
 {% if layout != 'home' %}
   <nav aria-label="secondary">
-    <ul>
-      {%- for item in primaryNav %}
-        {% if page.url | startsWith(item.url) %}
-          <li><a class="text-semibold" href="{{ item.url | url }}">{{ item.title }}</a></li>
+    <ul class="nav-secondary ul" data-nav-secondary>
+      {%- for item in primaryNav -%}
+        {%- if page.url | startsWith(item.url) -%}
+          <li class="nav-secondary__wrapper"><a class="nav-secondary__top" href="{{ item.url | url }}" data-nav-secondary-toggle>{{ item.title }}</a>
           {%- if item.children %}
-            {%- for subitem in item.children %}
-              <li>
-                <a href="{{ subitem.url | url }}" {% if page.url | startsWith(subitem.url) %} aria-current="page"{% endif %}>{{ subitem.title }}</a>
-              </li>
-            {%- endfor %}
+            <span class="nav-secondary__pipe">/</span>
```

Comment:
Thought: Could this be done in style and avoid the need to include it in code/hide it for small screens/remove semantically?

Reply:
It certainly can, all done 👍